Diary/Resource/Controller/DiaryItemViewController.swift

```diff
@@ -0,0 +1,239 @@
+        titleTextView.text = data.title
+        bodyTextView.text = data.body
+        title = DateFormatterManager().formatDate(data.createdAt)
+    }
+    
+    @objc private func showActionSheet() {
+        let alert: UIAlertController = UIAlertController(title: nil,
+                                                         message: nil,
+                                                         preferredStyle: .actionSheet)
+        
+        alert.addAction(UIAlertAction(title: Namespace.share, style: .default) { _ in
+            self.present(self.activityViewController, animated: true)
+        })
+        alert.addAction(UIAlertAction(title: Namespace.delete, style: .destructive) { _ in
+            let diaryItem = self.diaryItemManager.returnDiaryItem()
+            let handler: (UIAlertAction) -> Void = { _ in
+                self.delete(diaryItem)
+                self.navigationController?.popViewController(animated: false)
+            }
+            
+            self.showDeleteAlert(handler: handler)
+        })
+        alert.addAction(UIAlertAction(title: Namespace.cancel, style: .cancel))
+        
+        present(alert, animated: true)
+    }
+    
+    private func delete(_ diaryModel: DiaryModel?) {
+        self.titleTextView.text = Namespace.empty
+        self.bodyTextView.text = Namespace.empty
+        self.diaryItemManager.deleteDiary(data: diaryModel)
+    }
+    
+    private enum LayoutConstant {
+        static let stackViewSpacing = CGFloat(8)
+        static let mainStackViewTopMargin = CGFloat(8)
+        static let mainStackViewLeadingMargin = CGFloat(8)
+        static let mainStackViewBottomMargin = CGFloat(8)
+        static let mainStackViewTrailingMargin = CGFloat(8)
+        static let titleTextViewMaxHeight = CGFloat(100)
+    }
+}
+
+// MARK: dismiss keyboard
+extension DiaryItemViewController {
+    private func addKeyboardDismissAction() {
+        let swipeGesture: UISwipeGestureRecognizer = UISwipeGestureRecognizer(target: self,
+                                                                              action: #selector(dismissKeyboard))
+        swipeGesture.direction = .down
+        bodyTextView.addGestureRecognizer(swipeGesture)
+    }
+    
+    @objc private func dismissKeyboard() {
+        bodyTextView.resignFirstResponder()
+    }
+}
+
+extension DiaryItemViewController: UITextViewDelegate {
+    func textViewDidBeginEditing(_ textView: UITextView) {
+        if textView.textColor == .systemGray3 {
+            textView.text = nil
+            textView.textColor = .black
+        }
+    }
+    
+    func textViewDidChange(_ textView: UITextView) {
+        guard textView == titleTextView else { return }
+        
+        let titleTextViewHeight = textView.contentSize.height
+        
+        if titleTextViewHeight > LayoutConstant.titleTextViewMaxHeight {
+            isOversized = true
+        } else {
+            isOversized = false
+        }
```

Comment:
```suggestion
        isOversized = titleTextViewHeight > LayoutConstant.titleTextViewMaxHeight
```
컨디션에 따라 Bool 변수의 값이 바뀐다면 컨디션 자체를 넣어서 이렇게 바꿀수도 있어요.

Reply:
필요 없는 기능이라 삭제했습니다.